### ISSUE TITLE
The "Use Palette Registers" checkbox of the Bitmapped VRAM View doesn't work anymore

### ISSUE THREAD

**Issue by sdsnatcher** [openMSX/debugger#180]
The "Use Palette Registers" checkbox of the Bitmapped VRAM View doesn't work anymore. If I deselect it, the image is still shown with the current color palette.

Tested with the Build number 65, released this Apr 6, 2022.

**Comment by MBilderbeek** [comment]
I think the behaviour is different: if you deselect it, it uses the current palette which you can edit with the "Edit palette" option. It doesn't use the default palette.
Are you sure it ever worked differently?

Note that the release date of 'nightly' seems to be the date when this release got created. But the artifacts are updated whenever there are new commits :)

**Comment by sdsnatcher** [comment]
>  Are you sure it ever worked differently?

Yes, I used this feature extensively before. When the checkbox was disabled, the default V9938 palette was used instead. This helps a lot to debug some MSX software that set an all-black palette while uploading graphics to the VRAM.
(Yep, I know this is poor programing and they should've just disabled the screen, but that's how some bad programmers did it back then)

In such cases, switch to the "Edit palette" box would be very unproductive, as the palette is all black and the user would have to set colors manually there to se anything.

(Not to mention that the "Edit palette" box is glitchy and doesn't work correctly most of the time. But that would be a different bug report).

**Comment by MBilderbeek** [comment]
Please confirm this feature works fine in the built-in debugger. If not, please open a new issue in the openMSX/openMSX issue tracker.
